src/App.css

```diff
@@ -38,11 +38,20 @@
 .search {
   padding: 5px 0 20px 0;
 }
+tr{
+  border-top: 1px solid black;
 
-tr {
+}
+td{
```

Comment:
Looking good. This is a very small detail but for your `tr`, `th`, `td` selectors you should probably have a space between the selector and the opening curly bracket :)